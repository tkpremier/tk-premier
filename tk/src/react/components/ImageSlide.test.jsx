import React from 'react';
import ImageSlider from './ImageSlide';
import renderer from 'react-test-renderer';

it('should render an active slide', () => {
    const imageslider = renderer.create(
        <ImageSlider
            slideActive
            url="/auctions/auction/NY010317"
            imageUrl="https://8675fa3b12fbaae97df1-49af760ec3cce3b92480c70a3569b570.ssl.cf2.rackcdn.com/HomePage/slide-128.jpg"
        />
    );
    const tree = imageslider.toJSON();
    expect(tree).toMatchSnapshot();
});

it('should render an inactive slide', () => {
    const imageslider = renderer.create(
        <ImageSlider
            url="/auctions/auction/NY010317"
            imageUrl="https://8675fa3b12fbaae97df1-49af760ec3cce3b92480c70a3569b570.ssl.cf2.rackcdn.com/HomePage/slide-128.jpg"
        />
    );
    const tree = imageslider.toJSON();
    expect(tree).toMatchSnapshot();
});
