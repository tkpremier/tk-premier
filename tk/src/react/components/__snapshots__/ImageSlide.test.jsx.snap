// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render an active slide 1`] = `
<div
  className="phillips-slide active"
>
  <a
    href="/auctions/auction/NY010317"
    onClick={[Function]}
  >
    <div
      className="slide-image"
    >
      <div
        className="image-container"
        style={
          Object {
            "backgroundImage": "url(https://8675fa3b12fbaae97df1-49af760ec3cce3b92480c70a3569b570.ssl.cf2.rackcdn.com/HomePage/slide-128.jpg)",
            "backgroundPosition": "0% 0%",
            "backgroundSize": "cover",
          }
        }
      />
    </div>
  </a>
</div>
`;

exports[`should render an inactive slide 1`] = `
<div
  className="phillips-slide"
>
  <a
    href="/auctions/auction/NY010317"
    onClick={[Function]}
  >
    <div
      className="slide-image"
    >
      <div
        className="image-container"
        style={
          Object {
            "backgroundImage": "url(https://8675fa3b12fbaae97df1-49af760ec3cce3b92480c70a3569b570.ssl.cf2.rackcdn.com/HomePage/slide-128.jpg)",
            "backgroundPosition": "0% 0%",
            "backgroundSize": "cover",
          }
        }
      />
    </div>
  </a>
</div>
`;
